scripts/get_renv_file.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# if specified, first argument shoud be path to renv.lock local file, or a public URL to an renv.lock file
+# Script that get renv.lock path from file path or URL 
+
+renv="${1}"
+
+if [ -f "$renv" ]
+then
+    echo "renv file found locally"
+    cp "$renv" ./renv.lock
+elif [ ! -z "$renv" ]
+then
+    curl -LJO "$renv"
```

Comment:
I think this assumes that remote lockfile will be named `renv.lock` - maybe we could make this a little more robust to account for situations when the file is named differently.

Reply:
(@walkowif you're right about this, I add it to my todo list ! I just solve this thread since this is not a priority for now, but I'll make another branch on improving this action !)